cypress/e2e/home.cy.js

```diff
@@ -0,0 +1,16 @@
+/// <reference types="cypress" />
+
+import { loadMovies } from "../../src/movies.js";
+const url = "http://localhost:5080/";
+
+it("Check that all film titles are present and no more then 10 screenings are shown in the aside", async () => {
+    cy.visit(url);
```

Comment:
I see no benefit of having `url` be a variable declared elsewhere, over just having the `http://localhost:5080` value here: `cy.visit('http://localhost:5080')`.

When the value is elsewhere, it means I, the reader, must jump back and forth in the code. Sometimes that tradeoffs of having a very long URL or something might make that worthwhile, but here the URL is short and could easily fit inline without causing problems.